easybuild/easyconfigs/m/matplotlib/matplotlib-2.0.2-intel-2017a-Python-2.7.13.eb

```diff
@@ -16,17 +16,20 @@ exts_defaultclass = 'PythonPackage'
 
 dependencies = [
     ('Python', '2.7.13'),
-    ('freetype', '2.7.1'),
+    ('Tkinter', '%(pyver)s', versionsuffix),
+    ('freetype', '2.7.1', '-libpng-1.6.29'),
```

Comment:
@schiotz Please stick to `('freetype', '2.7.1')` here, to avoid introducing conflicts.

Reply:
@boegel I actually started with `('freetype', '2.7.1')`, but that caused a version conflict for libpng, which I got rid of in this way.